lmn/fixtures/fn_testing_users.json

```diff
@@ -1,10 +1,10 @@
 [
-    {"model" : "auth.user", "pk" : 1 , "fields" :
-      { "username" : "alice",
-        "first_name" : "alice",
-        "last_name" : "last",
-        "email" : "a@a.com",
-        "password" : "pbkdf2_sha256$30000$kQdWPwdzZfE3$IXwsyRPDGxrbRp7tb2wdRHnyDT0eTcZG2XWPnAbQcS4="
+    {"model" : "lmn.profile", 
```

Comment:
This fixture is used by the functional test. Create a new fixture file with this info in it, and restore the original content of this file